pmoves/tools/inject_github_pat_from_gh_cli.py

```
@@ -0,0 +1,161 @@
+        result = subprocess.run(
+            ["gh", "auth", "token"],
+            capture_output=True,
+            text=True,
+            check=True,
+            timeout=10,
+        )
+    except FileNotFoundError:
+        print("ERROR: gh CLI not found. Install: https://cli.github.com/", file=sys.stderr)
+        sys.exit(1)
+    except subprocess.CalledProcessError as e:
+        print(f"ERROR: gh CLI not authenticated. Run: gh auth login", file=sys.stderr)
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)